src/App.vue

```diff
@@ -61,6 +61,11 @@
     <ExitDialog />
     <ExitIncompletePaymentDialog />
     <HelpMeChooseDialog />
+    <IncorporateNowErrorDialog
+      attach="#app"
+      :dialog="incorporateNowErrorDialog"
+      @close="incorporateNowErrorDialog = false"
```

Comment:
What state does Close put the user in -- what can they do after this error?

Reply:
Back to square zero. They'll be going back to the main page and can try again from scratch.

Actually, now that you've mentioned this Sev, I might have to clear the session storage variable when the user closes the dialog. I'll be adding that.